src/transformers/data/processors/task_processors.py

```diff
@@ -0,0 +1,313 @@
+# coding=utf-8
+"""Data processors per task"""
+
+import copy
+import csv
+import json
+import logging
+from abc import ABC, abstractmethod
+
+from ...file_utils import is_tf_available, is_torch_available
+
+
+logger = logging.getLogger(__name__)
+
+
+class InputExample(object):
```

Comment:
Right, I copy/paste what there was in the `utils.py` file in case I had to update something without touching to what has been done in the framework. But definitely I will remove those.